add /opt/lib to default search path for the dynamic linker

--- a/Makeconfig
+++ b/Makeconfig
@@ -589,6 +589,9 @@ else
 default-rpath = $(libdir)
 endif
 
+# Add /opt/lib to default search path for the dynamic linker
+user-defined-trusted-dirs := /opt/lib
+
 ifndef link-extra-libs
 link-extra-libs = $(LDLIBS-$(@F))
 link-extra-libs-static = $(link-extra-libs)
